docs/examples/acp_client_stub.py

```
@@ -0,0 +1,123 @@
+        message = json.loads(line)
+        if message.get("id") == request.get("id"):
+            if message.get("error"):
+                raise RuntimeError(f"ACP error: {message['error']}")
+            return message
+        print(f"notification: {message}", file=sys.stderr)
+
+
+def main() -> int:
+    workspace = str(Path.cwd())
+    proc = subprocess.Popen(
+        ["v100", "acp"],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)